### DISCUSSION TITLE
Welcome to openWakeWord Discussions!

### DISCUSSION THREAD

**Discussion by dscripka** [dscripka/openWakeWord#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ccandaffio** [comment]
Trying to create a custom wake word, but when I enter the new word and hit play, I just get khum_puter. Any ideas?

**Reply by ccandaffio** [reply]
getting unpickeling error

**Comment by TrueAndyT** [comment]
I'm trying to make it work and it's all a bs show. No model can be loaded. tflite is not working - all is crap about this openwakeword. Frustrated AF
